frontend/app/openproject-app.js

```diff
@@ -30,7 +30,6 @@ var I18n = require('./vendor/i18n');
 
 // standard locales
 I18n.translations.en = require("locales/js-en.yml").en;
-I18n.translations.de = require("locales/js-de.yml").de;
```

Comment:
In the corresponding plugin PRs you are also removing the standard english locale, e.g. here: https://github.com/finnlabs/openproject-backlogs/pull/142

Probably I am missing a point here, but I'd expect those changes to look similar...